testsuite/cluster-test/src/instance.rs

```diff
@@ -271,6 +272,14 @@ impl Instance {
             )
             .await
     }
+
+    pub fn debug_interface_client(&self) -> AsyncNodeDebugClient {
+        AsyncNodeDebugClient::new(
+            self.http_client.clone(),
+            self.ip(),
+            self.debug_interface_port.unwrap_or_default() as u16,
```

Comment:
So returning default(which is `0`) here does not make much sense - debug client certainly won't be able to connect to port 0.
More reasonable would be to just crash here: `self.debug_interface_port.expect("debug_interface_port is not known on this instance")`